lib/adapter/freshheads/CssAdapter.ts

```diff
@@ -105,20 +106,24 @@ export default class CssAdapter implements Adapter {
         );
     }
 
-    private validateAllRequiredModulesAreInstalled() {
-        const requiredModules = [
-            'mini-css-extract-plugin',
-            'autoprefixer',
-            'css-loader',
-            'postcss-loader',
-        ];
-
-        requiredModules.forEach(module => {
-            if (!checkIfModuleIsInstalled(module)) {
-                throw new Error(
-                    `The '${module}' needs to be installed for this adapter to work`
-                );
-            }
+    private validateAllRequiredModulesAreInstalled(isProduction: boolean) {
+        const requiredModules: { [module: string]: string } = {
+            'mini-css-extract-plugin': '0.8.0',
+            autoprefixer: '9.7.0',
+            'css-loader': '3.2.0',
+            'postcss-loader': '3.0.0',
+        };
+
+        if (isProduction) {
```

Comment:
Het lijkt mij niet zo handig om deze module  alleen te valideren wanneer je de productie build maakt. Wat je dan krijgt is dat je pas in een volgende stap er achter komt dat je dependencies mist terwijl we weten dat je uiteindelijk toch wel een productie build gaat maken.

Reply:
Ja, dat is wel een goed punt inderdaad. Zal hier nog even naar kijken.

Reply:
https://github.com/freshheads/webpack-config-builder/pull/43/commits/f9863af9293f888ad4703f4e1c37ff609386fb79